src/Model/Behavior/AclBehavior.php

```diff
@@ -113,7 +114,12 @@ public function node($ref = null, $type = null)
             throw new Exception\Exception(__d('cake_dev', 'ref parameter must be a string or an Entity'));
         }
 
-        return $this->_table->{$type}->node($ref);
+        if(property_exists($this->_table, $type)){
```

Comment:
```suggestion
        if (property_exists($this->_table, $type)){
```

You'll need to apply the code formatting standards. You can do that with `vendor/phpcbf src tests`